test/service/test_manifest.py

```diff
@@ -73,7 +74,7 @@ def tearDown(self):
 
     def _setup_git_commit(self):
         """
-        Set git variables required to generate the object key of format='full' type manifests
+        Set git variables required to generate the object key of full-type manifests
```

Comment:
```suggestion
        Set git variables required to derive the manifest object key
```
Not your fault, but when originally added this was only needed for the test of full format manifests. Now things have changed a bit and the object key is used in the generation of all manifests.